### PR TITLE
[14.0][IMP] auditlog, add auditlog.log.line view

### PR DESCRIPTION
**Pull request by kittiu**
Add auditlog.log.line menu.

### CODE REVIEW

**Review by gurneyalex — APPROVED** [review]

File: auditlog/views/auditlog_view.xml [hunk 1]
```diff
@@ -257,4 +257,99 @@
...
+    <record id="action_auditlog_line" model="ir.actions.act_window">
+        <field name="name">Auditlog Report</field>
+        <field name="res_model">auditlog.log.line</field>
+        <field name="view_mode">tree</field>
+        <field name="search_view_id" ref="view_auditlog_line_search" />
+        <field name="context">{'search_default_group_by_model_id': 1}</field>
+    </record>
+    <menuitem
+        id="menu_auditlog_line"
+        name="Logs Report"
```
Comment (yostashiro) [inline auditlog/views/auditlog_view.xml]:
```suggestion
        name="Log Lines"
```
My personal preference.

File: auditlog/views/auditlog_view.xml [hunk 2]
```diff
@@ -257,4 +257,99 @@
...
+        parent="menu_audit"
+        action="action_auditlog_line"
+        sequence="20"
```
Comment (yostashiro) [inline auditlog/views/auditlog_view.xml:353]: This moves the menu item at the bottom inside 'Audit'.  Could we perhaps add the sequences 30 and 40 to 'User sessions' and 'HTTP Requests', which currently do not seem to have a sequence specified?

image

File: auditlog/views/auditlog_view.xml (hunk 1 above)
Comment (bodedra) [reply auditlog/views/auditlog_view.xml]: +1

File: auditlog/views/auditlog_view.xml (hunk 2 above)
Comment (bodedra) [reply auditlog/views/auditlog_view.xml:353]: +1

Comment (kittiu) [reply auditlog/views/auditlog_view.xml:353]: Done, thanks for your suggestion.

**Review by yostashiro — APPROVED** [review]
LGTM 👍  Thanks, @kittiu !

File: auditlog/views/auditlog_view.xml [hunk 3]
```diff
@@ -257,4 +257,99 @@
         parent="menu_audit"
         action="action_auditlog_log_tree"
     />
+    <!-- auditlog.log.line -->
+    <record model="ir.ui.view" id="view_auditlog_line_tree">
+        <field name="name">view.auditlog.line.tree</field>
+        <field name="model">auditlog.log.line</field>
+        <field name="arch" type="xml">
+            <tree string="Auditlog Report">
```
Comment (bodedra) [inline auditlog/views/auditlog_view.xml]:
```suggestion
            <tree string="Log Lines">
```

File: auditlog/views/auditlog_view.xml [hunk 4]
```diff
@@ -257,4 +257,99 @@
...
+                <field name="new_value_text" optional="show" />
+                <field name="log_type" optional="hide" />
+            </tree>
+        </field>
+    </record>
+    <record id="view_auditlog_line_search" model="ir.ui.view">
+        <field name="name">auditlog.line.search</field>
+        <field name="model">auditlog.log.line</field>
+        <field name="arch" type="xml">
+            <search string="Logs Report">
```
Comment (bodedra) [inline auditlog/views/auditlog_view.xml]:
```suggestion
            <search string="Log Lines">
```

File: auditlog/views/auditlog_view.xml [hunk 5]
```diff
@@ -257,4 +257,99 @@
...
+                        string="HTTP Request"
+                        domain="[]"
+                        context="{'group_by':'http_request_id'}"
+                    />
+                </group>
+            </search>
+        </field>
+    </record>
+    <record id="action_auditlog_line" model="ir.actions.act_window">
+        <field name="name">Auditlog Report</field>
```
Comment (bodedra) [inline auditlog/views/auditlog_view.xml]:
```suggestion
        <field name="name">Log Lines</field>
```

**Review by bodedra — CHANGES_REQUESTED** [review]
I did functional review and LGTM :+1: 

Minor correction on naming.

Change needed:

- If you notice, we can not create "Log" manually. So let's do same thing for "Log Lines"

Screenshot 2021-08-25 at 08-21-30 Auditlog Report - Odoo

File: auditlog/views/auditlog_view.xml (hunk 5 above)
Comment (kittiu) [reply auditlog/views/auditlog_view.xml]: Done, thank you!

File: auditlog/views/auditlog_view.xml [hunk 6]
```diff
@@ -257,4 +257,99 @@
         parent="menu_audit"
         action="action_auditlog_log_tree"
     />
+    <!-- auditlog.log.line -->
+    <record model="ir.ui.view" id="view_auditlog_line_tree">
+        <field name="name">view.auditlog.line.tree</field>
+        <field name="model">auditlog.log.line</field>
+        <field name="arch" type="xml">
+            <tree string="Log Lines">
```
Comment (bodedra) [inline auditlog/views/auditlog_view.xml]:
```suggestion
            <tree string="Log Lines" create="false">
```

**Review by bodedra — CHANGES_REQUESTED** [review]
Create button still visible.

auditlog

I added suggestion to fix it.

**Review by bodedra — APPROVED** [review]
Functional review :+1:

**Pull request merged**